oaknational/Oak-Web-Application
# [CUR-185] Fix examboard slug

## Description

- Added fixtures
- Updated data for each page / component
- Pulled in correct slug to curriculum header 
- Updated tests

## How to test

1. Go to https://deploy-preview-1797--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum/
2. Click on Secondary, English and then Edexcel
3. You should see `/english-secondary-edexcel` in the browser as the slug


## Checklist
